### PR TITLE
Upgrade and Fix: Zoom Free Calling Module

### PR DESCRIPTION
**Pull request by cbshoaib**
# Updated-Readme: Zoom Calling Module

## Type of PR

- [ ] Bugfix
- [ ] New feature
- [x] Minor changes

## Changes introduced
added module instructions, credientials setup, scope features, module specifications, contributing and license  in README.md file.

### CODE REVIEW

File: modules/react-native-zoom-free-calling/store/api.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import axios from "axios";
+import options from "../options";
+import base64 from "react-native-base64";
+
+const BASE_URL = "https://api.zoom.us";
```
Comment (danielsousaio) [inline modules/react-native-zoom-free-calling/store/api.js:5]: Can we make this a module option?

**Review by danielsousaio — COMMENTED** [review]
Needs updated readme

File: modules/react-native-zoom-free-calling/store/api.js (hunk 1 above)
Comment (saad-abid-crowdbotics) [reply modules/react-native-zoom-free-calling/store/api.js:5]: This is the base URL for Zoom APIs, this will be the same for everyone.

**Review by saad-abid-crowdbotics — APPROVED** [review]
LGTM

**Pull request merged**